source/includes/stackpath/_workloads.md

```diff
@@ -117,6 +117,103 @@ Attributes | &nbsp;
 `isRemoteManagementEnabled`<br/>*boolean* | Specify if remote management is enabled on workload instance or not.
 `image`<br/>*string* | The workload's instance operating system image.
 
+<!-------------------- CREATE A WORKLOAD -------------------->
+### Create a workload
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -d "request_body" \
+    "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads"
+```
+> Request body example for a VM:
+```json
+{
+   "name":"w-user-zwg",
+   "slug":"w-user-zwg",
+   "type":"VM",
+   "image":"stackpath-edge/centos-7-cpanel:v201905241955",
+   "vpc":"Default",
+   "addAnyCastIpAddress":false,
+   "publicPorts":"80",
+   "protocol":"TCP",
+   "firstBootSshKey":"ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDcYr9OnzsDfYVW2I1kX/iYJ0mPG490bI5mbxbOAKPLMuWLguxRohX804j1XbwZJ+Sna+9rSfxaYA8vgd1MoYX10l9cnMLx/MMbYp4ZquauN4pGY3WoDeCqsTss3VUMW+7RFBILpU3SJTlDV02FI36D3IXb4A8XymCyU3KC99XXTfTQsuKC+WFRMsTWtklrasqCVd5yEG90i/aJc6A3TZGOYgPFNEeVYvNDaJmIkb3y4FfShoBIMgZRt0ay7SvWZUvyfvyNmK5W9ePdhZZ58R+7tQNmCzjQ4v0suWRuGJ/XL3+03w3HEsDdQx+noL+R+qAjoNFwc0spBBhJK+Q4ADqr nothing@gmail.com",
+   "specs":"SP-1",
+   "persistenceStoragePath":"",
+   "persistenceStorageSize":1,
+   "deploymentName":"wi-root-ion",
+   "deploymentInstancePerPops":1,
+   "enableAutoScaling":false,
+   "deploymentPops":"YYZ"
+}
+```
+> Request body example for a Container:
+```json
+{
+   "name":"w-user-pah",
+   "type":"CONTAINER",
+   "image":"nginx:latest",
+   "addImagePullCredentialsOption":true,
+   "containerUsername":"test.username",
+   "containerPassword":"test-password",
+   "containerServer":null,
+   "containerEmail":null,
+   "environmentVariableKey": null,
+   "environmentVariableValue": null,
+   "secretEnvironmentVariableKey":null,
+   "secretEnvironmentVariableValue":null,
+   "vpc":"Default",
+   "addAnyCastIpAddress":false,
+   "publicPorts":"80-200",
+   "protocol":"UDP",
+   "specs":"SP-3",
+   "persistenceStoragePath":null,
+   "persistenceStorageSize":1,
+   "deploymentName":"wi-root-pxa",
+   "deploymentInstancePerPops":1,
+   "deploymentPops":"FRA",
+   "enableAutoScaling":true,
+   "cpuUtilization":50,
+   "minInstancesPerPop":1,
+   "maxInstancesPerPop":2
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/workloads</code>
+
+Create a new workload
```

Comment:
Should be `Create a new workload in a given [environment](#administration-environments).`

Reply:
good catch. Thanks Pat